### DISCUSSION TITLE
what's next

### DISCUSSION THREAD

**Discussion by unknown07724** [TurboWarp/extensions#1629] [General]
a xbox and playstation integration extension

**Comment by unknown07724** [comment]
![conosle-banner](https://github.com/user-attachments/assets/1152b3a9-5106-447c-a3ec-98c5ff79b4f5)
the extension banner

**Comment by WAYLIVES** [comment]
nice

**Comment by unknown07724** [comment]
This has been canceled for the mean time, bc how the fuck to port C++ and C# into JS

**Reply by Brackets-Coder** [reply]
Number 1, chill the absurd language. It's not tolerated at all in this repository (or at least by me, but I'm pretty sure @yuri-kiss had to tell someone else too, although I may be confused with someone else). Number two, you're probably going to have a lot of trouble getting proprietary SDKs from Microsoft and Sony, but if you can, try using something like Emscripten, which was used to compile Ammo.js from the C++ Bullet Physics SDK. I'm writing an Ammo.js extension and it works pretty much exactly like Bullet

**Reply by unknown07724** [reply]
that is why I canceled it

**Reply by yuri-kiss** [reply]
> Number 1, chill the absurd language. It's not tolerated at all in this repository (or at least by me, but I'm pretty sure @yuri-kiss had to tell someone else too, although I may be confused with someone else). Number two, you're probably going to have a lot of trouble getting proprietary SDKs from Microsoft and Sony, but if you can, try using something like Emscripten, which was used to compile Ammo.js from the C++ Bullet Physics SDK. I'm writing an Ammo.js extension and it works pretty much exactly like Bullet

I have told people off for saying abuse stuff towards others yes

> that is why I canceled it

yeah and even then it wouldnt be possible to communicate with the SDK, and the SDK would require access to the hardware which the project cant run on (AT ALL)

**Reply by unknown07724** [reply]
I was an idiot and still am
